tendrl/commons/flows/base_flow.py

```diff
@@ -156,18 +156,19 @@ def extract_atom_details(self, atom_name):
         namespace = atom_name.split('.objects.')[0]
         object_name = atom_name.split('.objects.')[1].split('.atoms.')[0]
         atoms = self.definitions[namespace]['objects'][object_name]['atoms']
-        atom = atoms[atom_name.split('.')[-1]]
+        atom = atoms[atom_name.split('.')[-2]]
```

Comment:
I am ok with anything. The point is we are mixing uppercase and lowercase a lot for object and atom names. I personally feel we should restrict to uppercase for only object names and keep atom names lowercase only and as in above fix we are actually getting the module name for the atom details to be loaded from definitions.
@r0h4n @brainfunked @nthomas-redhat comments?